frontend/src/metabase-lib/parameters/utils/parameter-values.unit.spec.js

```diff
@@ -78,7 +78,7 @@ describe("parameters/utils/parameter-values", () => {
   });
 
   describe("getValuePopulatedParameters", () => {
-    it("should return an array of parameter objects with the `value` property set if it exists in the given `parameterValues` id, value map", () => {
+    it("should return an array of parameter objects with the `value` property set if it exists in the given `parameterValues` id, value map, and null if it doesn't exist", () => {
```

Comment:
nit: maybe it worth having a separate test for `null`

Reply:
Why? What's the principle I should apply here for creating a separate test for this?

Reply:
Never mind my questioning, I tried it out and I believe it is easier to read with a separate test db0eca2e826c67f0e7c3b889cc673fbedc5134ba

Reply:
overall it's better to have one thing tested at once and I found it's a case here